src/other/acr/acr-en.hbs

```diff
@@ -0,0 +1,323 @@
+
+<p>The method consists of the following phases:</p>
+<ol>
+	<li><strong>Producing assessments</strong> involves evaluating a test subject based on a set of Accessibility Testing Rules (ACT).
+</li>
+	<li><strong>Maintaining a living accessibility conformance report (ACR)</strong>, until <em>no further testing</em> is required to establish conformance or until the test subject undergoes a major backward incompatible change that invalidates some or all related assessments included in the ACR.</li>
+	<li><strong>Issuing the ACR</strong>, indicating when the report is final and represents the state of the test subject. When issued, the ACR shall not reference any additional assessments or be edited except for archiving.</li>
+	<li><strong>Archiving the ACR</strong>, indicating that the report does not represent the current accessibility state of the test subject and suggesting that a new ACR report should be produced with a set of new assessments.</li>
+</ol>
+
+<h2>Type of documents</h2>
+<p>There are two major types of documents: <strong>assessments</strong>, which are considered <em>informal</em>, and <strong>accessibility conformance reports</strong>, which are considered <em>formal</em>. This means that while assessments may report various states about success criteria testing (fail, pass, not applicable), these results may be audited differently for accuracy, false positives, or false negatives when producing the accessibility conformance report.</p>
+
+<div class="row wb-eqht-grd">
+	<div class="col-md-6">
+		<section class="well well-lg">
+			<h3 class="mrgn-tp-0">Assessments</h3>
+			<p>Informal report</p>
+			<p>Outcome:</p>
+			<ul>
+				<li>Pass</li>
+				<li>Fail</li>
+				<li>Inapplicable</li>
+			</ul>
+			<p>Evaluate <a href="https://www.w3.org/TR/EARL10-Schema/#TestCase">test cases</a></p>
+			<p>Leverage a set of <em><a href="https://www.w3.org/WAI/standards-guidelines/act/">Accessibility conformance testing (ACT) rules</a></em> to establish the result.</p>
+		</section>
+	</div>
+	<div class="col-md-6">
+		<section class="well well-lg">
+			<h3 class="mrgn-tp-0">Accessibility conformance report (ACR)</h3>
+			<p>Formal report when there is an issued date</p>
+			<p>Outcome:</p>
+			<ul>
+				<li>Satisfied</li>
+				<li>Not satisfied</li>
+				<li>Further testing is needed</li>
+			</ul>
+			<p>Evaluate <a href="https://www.w3.org/TR/EARL10-Schema/#TestRequirement">test requirements</a></p>
+			<p>Leverage a set of <em>assessments</em> to establish the conformity.</p>
+		</section>
+	</div>
+</div>
+
+<h2>Theoretical complete example</h2>
+<div class="row wb-eqht-grd">
+	<div class="col-md-6">
+		<section class="well well-lg">
+			<h3 class="mrgn-tp-0">Assessments</h3>
+			<ul class="lst-spcd list-unstyled">
```

Comment:
Add one assessment example (pre-assessment) where the assessment consist into identifying the applicability of the SC and with an overall indication of the required expertise to perform a full assessment.

Adjust the HTML template accordingly.

Might require the following new property on the `earl:Assertion`: `acr:requiresExpertise` where it can contains a literal or a reference which consist at least into the following pre-defined value:
* aria : When the test subject require someone that has a good understanding of WAI-ARIA specification and related design pattern
* html : When the test subject require someone that has a good understanding of HTML specification
* form : When the test subject require someone that has a good understanding of web form design
* audioVideo: When the test subject require someone that has a good understanding of audio/video content
* imaging: When the test subject require someone that has a good understanding of imaging and non-text content

Reply:
J'ai définie une nouvelle propriété nommé "acr:involvesExpertise" qui est d'une nature plus suggestive. il pourra contenir une valeur littéral ou une valeur prédéfinie du type "acr:Topic" d'où les instances sera disponible dans cet espace: "acr:topics#identifiantDuSujet"